src/test/javascript/spec/component/plagiarism/plagiarism-split-view.component.spec.ts

```diff
@@ -128,14 +155,47 @@ describe('Plagiarism Split View Component', () => {
         expect(textSubmissionB).toEqual(submissionB);
     });
 
-    it('parses text matches', () => {
+    it('should parse text matches', () => {
         jest.spyOn(comp, 'mapMatchesToElements').mockReturnValue(new Map());
 
-        const matches: PlagiarismMatch[] = [];
+        const matches: PlagiarismMatch[] = [
+            { startA: 0, startB: 0, length: 5 },
+            { startA: 10, startB: 10, length: 20 },
+        ];
         comp.parseTextMatches({ submissionA, submissionB, matches } as PlagiarismComparison<TextSubmissionElement>);
 
         expect(comp.matchesA).toBeDefined();
```

Comment:
Using `toBeDefined()` is violating our client test guidelines. Is it possible to specify the expectation here?

Reply:
@willergeorg Good catch, I must have missed that. I changed it now to a test that makes more sense 👍🏼